docs/walkthrough-1.6.md

```diff
@@ -454,7 +277,7 @@ ups-instance-credential           Opaque                                2
 
 Notice that a new `Secret` named `ups-instance-credential` has been created.
 
-## Step 10 - Unbinding from the ServiceInstance
+# Step 6 - Unbinding from the ServiceInstance
```

Comment:
We should probably not call this unbinding anymore.  How about 'delete the ServiceInstanceCredential', and update the language to be something like 'Imagine we're done using this service instance.  We need to delete the credentials we requested earlier.'

Reply:
done